numpy/f2py/src/fortranobject.h

```diff
@@ -13,18 +13,26 @@ extern "C" {
 #include "numpy/arrayobject.h"
 #include "numpy/npy_3kcompat.h"
 
-
 #ifdef F2PY_REPORT_ATEXIT
 #include <sys/timeb.h>
-  extern void f2py_start_clock(void);
-  extern void f2py_stop_clock(void);
-  extern void f2py_start_call_clock(void);
-  extern void f2py_stop_call_clock(void);
-  extern void f2py_cb_start_clock(void);
-  extern void f2py_cb_stop_clock(void);
-  extern void f2py_cb_start_call_clock(void);
-  extern void f2py_cb_stop_call_clock(void);
-  extern void f2py_report_on_exit(int,void*);
```

Comment:
Actually prefer this a bit, but will defer to the formatter  in this case.

Reply:
Me too, I'm not sure which rule is triggering this.

Reply:
`AlwaysBreakAfterReturnType: TopLevel`. Try `TopLevelDefinitions` instead.

Reply:
Note that `// clang-format off` and `// clang-format on` can be used in the code to mask regions.